src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Rules.cshtml

```diff
@@ -7,7 +8,7 @@
         <div class="row">
             <div class="col-md-12">
                 <div class="text-center">
-                    <h2 class="mb-30">Regulamin</h2>
+                    <h2 class="mb-30">@Policy</h2>
```

Comment:
use `TermsOfUse`

Reply:
There are similar words like: Regutlamin, Polityka Prywatności so I changed to:

@PrivacyPrivacy -> Polityka Prywatnosci
@TermsOfUse -> Regulamin
@AcceptTermsOfUse -> Zostawiając tutaj swojego maila zgadzasz się na postanowienia regulaminu dostępnego tutaj

Reply:
@PrivacyPolicy *